task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
@timoreimann, I wanted to to this
Its a quote form related issue #111 

> When you're trying to create marathon application, it will split your AppID into segments by "/", and then check if every segment is matching 
^(([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])\\.)*([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])$|^(\\.|\\.\\.)$
> 
> I've been thinking about implementing the same AppID validation in go-marathon library, so it can be run in app.Name() method and Application constructors. 
We have utils.validateID(), but it is not really validating ID.
I suggest changing
validateID(string) string
to 
validateID(string) (bool, string) and add actual validation in it.

It seem's that other methods working fine. But I'll check it out once more.